docs/vaahstore/index.md

```diff
@@ -0,0 +1,51 @@
+# VaahStore
+
+[[toc]]
+
+## Introduction
+
+VaahStore is a E-Commerce framework built on top of the latest opensource technologies such as `Laravel 10`, a `PHP` framework and `Vue 3`, a progressive Javascript framework.
+
+It is suitable for all small or large E-Commerce business demands using a simple set-up procedure. Built on top of `VaahCms` and equipped with an easy product information management.
+
+- Multi-vendor
+- Multi-currency
+- Multi-lingual
+- Multi Country Taxes
+- Wishlists
+- Headless Product Content
+- Waitlist
+- Brand Store
+- Abandoned Cart
+- Coupon
+- AR Products Assets
+- Subscriptions
+- Dropshipping
+- Affiliates
+
+## Requirements
+
+### Minimum System Requirements
```

Comment:
@bharatkumar-j001 this is also incorrect, write correct PHP version, vaahcms, version, node and npm version